public/app/features/query/state/PanelQueryRunner.ts

```diff
@@ -105,97 +106,104 @@ export class PanelQueryRunner {
     }
 
     return this.subject.pipe(
-      this.getTransformationsStream(withTransforms),
-      map((data: PanelData) => {
-        let processedData = data;
-        let streamingPacketWithSameSchema = false;
+      mergeMap((data: PanelData) => {
+        if (data.series === lastRawFrames) {
+          console.log('returning same series');
+          return of({ ...data, structureRev, series: lastProcessedFrames });
+        }
```

Comment:
The issue i discovered is caused by this early return. Think we also need to compare field configs and update lastProcessedFrames accordingly here.

Reply:
or perhaps make sure that mutations to the frame also replace the frame container array 🤔